exercises/all-your-base/test/test_all_your_base.c

```diff
@@ -0,0 +1,200 @@
+#include "vendor/unity.h"
+#include <stddef.h>
+#include <stdint.h>
+#include "../src/all_your_base.h"
+
+#include <stdio.h>
+
+void fill_array(int16_t data[], size_t data_length, int16_t array[],
+                size_t array_length)
```

Comment:
One thing that might allow us to keep the tests and implementation simple is to let `digits[0]` be the ones digit, not the most significant digit. This means that the digits will look "backwards" in the tests, but the digits will be in the correct order and placement in the tests without resorting to a reformatting helper and the implementation will not need to know the number of resulting digits ahead of time to properly format the output. As a bonus, the index will match the exponent of the base for the digit.

Thoughts?